app/server/utils/slack_controller.py

```diff
@@ -392,17 +442,20 @@ def slack_controller(payload):
         submission = payload['submission']
         kyc = user.kyc_applications[0]  # most recent
 
-        kyc.first_name = submission['first_name']
-        kyc.last_name = submission['last_name']
-        kyc.dob = submission['date_of_birth']
-        kyc.street_address = submission['address']
+        kyc.first_name = submission['first_name'] or kyc.first_name
+        kyc.last_name = submission['last_name'] or kyc.last_name
+        kyc.dob = submission['date_of_birth'] or kyc.dob
+        kyc.street_address = submission['address'] or kyc.street_address
```

Comment:
is submission guaranteed to `None` by default? otherwise, submission.get('first_name', None) might be more reliable

Reply:
good point, `submission.get('first_name', None)` is alot cleaner! will update

Reply:
Well actually, it's not none by default, basically only want to update if defined....

`submission.get('first_name', kyc.first_name)` works